README.md

```diff
@@ -0,0 +1,2 @@
+# DanBotHostingStats
+All in one monitoring software to monitor multiple systems from a easy to use web panel
```

Comment:
1. this is more of a discord bot
2. barely monitors anything
3. web panel developed by someone else
4. jesus christ fix your grammar